d2/src/main/java/com/linkedin/d2/balancer/simple/SimpleLoadBalancer.java

```diff
@@ -890,6 +929,44 @@ public void getLoadBalancedClusterAndUriProperties(String clusterName, boolean w
     }
   }
 
+  public void processGetLoadBalancedClusterAndUriPropertiesTimeoutError(
+          Throwable e, String clusterName,
+          Callback<Pair<ClusterProperties, UriProperties>> clusterAndUriPropertiesCallback)
+  {
+    Pair<ClusterProperties, UriProperties> pair = getClusterAndUriPropertiesFromCache(clusterName,
+            clusterAndUriPropertiesCallback);
+    if (pair != null)
+    {
+      clusterAndUriPropertiesCallback.onSuccess(pair);
+    }
+    else
+    {
+      clusterAndUriPropertiesCallback.onError(new ServiceUnavailableException(clusterName,
+              "PEGA_1011. " + e.getMessage(), e));
+    }
+  }
+
+  // Get the cluster and uri properties from the cache, like INDIS, FStore(highest priority).
+  // If the properties are not found, call the callback with an error.
+  private Pair<ClusterProperties, UriProperties> getClusterAndUriPropertiesFromCache(String clusterName,
+                                                                                     Callback<Pair<ClusterProperties,
+                                                                                             UriProperties>> clusterPropertiesCallback)
```

Comment:
Idk what happened here but definitely needs fixing!

Reply:
My intelliji default format did this. I will reformat it mannully.